sockets/twitch.js

```diff
@@ -252,6 +268,31 @@ module.exports = function(io) {
             });
         }
 
+        function processFollowers() {
+            console.log('Processing new followers...');
+            /*var dummyFollower = {
+                user:{
+                    name:'Dummy Follower'
+                }
+            }
+
+            currentFollowers.unshift(dummyFollower);
+            console.log(currentFollowers);*/
+            var numNewFollowers = twitchData.twitchFollowers - numFollowersAtLaunch;
```

Comment:
Does `numFollowersAtLaunch` eventually need to get set to `numNewFollowers`? It seems it would keep displaying the same followers.